shiny/playwright/controller/_controls.py

```diff
@@ -5820,8 +5861,45 @@ def __init__(self, page: Page, id: str) -> None:
             loc="> li.nav-item",
         )
 
+    def expect_placement(
+        self, location: Literal["above", "below"] = "above", *, timeout: Timeout = None
+    ) -> None:
+        """
+        Expects the navset card underline to have the specified placement.
 
-class NavsetHidden(_NavPanelBase):
+        Parameters
+        ----------
+        location
+            The expected placement location. Defaults to `'above'`.
+        timeout
+            The maximum time to wait for the expectation to pass. Defaults to `None`.
+        """
+        if location == "below":
+            playwright_expect(self.loc_container.locator("..")).to_have_class(
+                "card-footer", timeout=timeout
+            )
+        else:
+            playwright_expect(self.loc_container.locator("..")).to_have_class(
+                "card-header", timeout=timeout
+            )
+
+    def expect_sidebar(self, sidebar: bool, *, timeout: Timeout = None) -> None:
```

Comment:
Please add a `.loc_sidebar` property to all classes that contain `.expect_sidebar()`